### PR TITLE
Fix traceback because _seen_ops is not initialised

### PR DESCRIPTION
**Pull request by barry-scott**
must call the base class __init__

### CODE REVIEW

**Comment by Byron** [comment]
Argh, thanks ! That code is not called by the test-suite yet :( .

File: git/util.py [hunk 1]
```diff
@@ -320,10 +320,10 @@ class CallableRemoteProgress(RemoteProgress):
     
     def __init__(self, fn):
         self._callable = fn
-        
+        RemoteProgress.__init__(self)
```
Comment (nvie) [inline git/util.py]: This should be `super(CallableRemoteProgress, self).__init__()` instead.

Comment (Byron) [inline git/util.py]: Thanks - it's done.

**Comment by nvie** [comment]
Cheers!

**Pull request merged**